keps/sig-apps/1591-daemonset-surge/README.md

```diff
@@ -328,18 +309,10 @@ details). For now, we leave it here.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How does this feature react if the API server and/or etcd is unavailable?**
+This feature will not work if the API server or etcd is unavailable as the controller-manager won't be even able get events or updates for StatefulSets.
```

Comment:
What happens if it's mid-rollout and this occurs? How can someone handle that failure?